### DISCUSSION TITLE
Adding Global Github Workflow

### DISCUSSION THREAD

**Discussion by Priyansh61** [bsoc-bitbyte/.github#1] [Ideas]
Hi Folks,

I guess it would be a good idea to setup, global workflows for all the repos instead of using individual workflow.

So can you please mention which all workflow we require, that needs to be setup?

Let me know your ideas below.

Thanks,

cc @prajjwalkapoor @arminpatel @varun7singh @sambhavgupta0705 @ChaudharyRaman @bsoc-bitbyte/maintainers-forum

**Comment by VarunGitGood** [comment]
True greetings lint and formatting along with commit message could be some I could think of right now for global but individual tests are a different matter

**Comment by sambhavgupta0705** [comment]
Three workflows would like to propose:
- Lint PR name : https://github.com/JulienKode/pull-request-name-linter-action
- Greeting workflow: https://github.com/actions/starter-workflows/blob/main/automation/greetings.yml
- Merge conflict : https://github.com/marketplace/actions/merge-conflict-finder

**Reply by prajjwalkapoor** [reply]
Yeah, it's nice to have these as global workflows.

**Reply by ankitpatil7** [reply]
really great!
